apps/server/src/modules/room/api/room.uc.ts

```diff
@@ -2,10 +2,9 @@ import { ConfigService } from '@nestjs/config';
 import { Injectable } from '@nestjs/common';
```

Comment:
You might want to revist room.uc.spec.ts and check if we really don't need any uc tests for the added methods.
I'm not sure if they are required. I'm not a fan of putting tests without a need.